vulnerabilities/importers/elixir_security.py

```diff
@@ -74,23 +71,18 @@ def collect_packages(self):
         packages = set()
         files = self._updated_files.union(self._added_files)
         for f in files:
-            with open(f) as file:
-                data = yaml.safe_load(file)
-                if data.get("package"):
-                    packages.add(data["package"])
+            data = load_yaml(f)
+            if data.get("package"):
+                packages.add(data["package"])
 
         return packages
 
-    def generate_all_version_list(self, pkg_name):
+    def get_versions_from_range(self, version_list, pkg_name):
+        pkg_versions = []
         if not getattr(self, 'pkg_manager_api', None):
```

Comment:
What's the purpose of this ? Even if  it is set this way, it will be useless since it's  not being loaded  with data right ?

Reply:
Ok I will remove it